cli/da-cli-script.py

```diff
@@ -0,0 +1,349 @@
+#!/usr/bin/env python
+import argparse
+import requests
+import sys
+
+GAV = "GROUP_ID:ARTIFACT_ID:VERSION"
+GA  = "GROUP_ID:ARTIFACT_ID"
+PRODUCT_VERSION = "PRODUCT_NAME:VERSION"
+PRODUCT_VERSION_STATUS = "PRODUCT_NAME:VERSION:SUPPORT_STATUS"
+STATUS_VALUES   = ['SUPPORTED', 'UNSUPPORTED', 'SUPERSEDED', 'UNKNOWN']
+
+def verify_response(response, failure_msg):
+    if 'success' in response and not response['success']:
+        print(failure_msg)
+    if 'message' in response:
+        print(response['message'])
+
+def helper_print_white_artifacts_products(response, show_artifacts=True):
+    if type(response) == dict and response['message']:
+        print(response['message'])
+    else:
+        for item in response:
+            product = item['name']
+            product_version = item['version']
+            product_support = item['supportStatus']
+
+            if show_artifacts:
+                gid = item['gav']['groupId']
+                aid = item['gav']['artifactId']
+                ver = item['gav']['version']
+                print("{}:{}:{} \t {}:{}:{}".format(gid, aid, ver, product,
+                                                    product_version, product_support))
+            else:
+                print("{}:{}:{}".format(product, product_version, product_support))
```

Comment:
Should the support be separated by ``:`` from ``PRODUCT:VERSION``? Wouldn't be space/tab better?